.evergreen/config.yml

```diff
@@ -2431,6 +2431,20 @@ buildvariants:
   display_name: "PyOpenSSL ${platform} ${python-version-windows} ${auth}"
   tasks:
     - '.replica_set'
+  rules:
+    # Python 3.10 does not support TLS on MongoDB <= 3.4.
```

Comment:
It's more like "On Windows, Python 3.10+ and MongoDB <= 3.4 don't have any ciphers in common."

Reply:
Done.